helix-core/src/main/java/org/apache/helix/PropertyPathBuilder.java

```diff
@@ -112,6 +115,12 @@
         "/{clusterName}/INSTANCES/{instanceName}/CURRENTSTATES/{sessionId}/{resourceName}");
     addEntry(PropertyType.CURRENTSTATES, 5,
         "/{clusterName}/INSTANCES/{instanceName}/CURRENTSTATES/{sessionId}/{resourceName}/{bucketName}");
+    addEntry(PropertyType.CUSTOMIZEDSTATES, 2,
```

Comment:
You probably don't need this since you build the path using instanceCustomizedState

Reply:
Actually this one is not deprecated. It's used in initialization. If I remove this one, the path is not created.